COLLADAMaya/src/COLLADAMayaDocumentExporter.cpp

```diff
@@ -454,9 +454,9 @@ namespace COLLADAMaya
     {
         // Make an unique COLLADA Id from a dagPath.
         // We are free to use anything we want for Ids. For now use
-        // a honking unique name for readability - but in future we
+        // full path name for readability - but in future we
```

Comment:
Please update comment based on email discussion
Provide insight why full path is necessary and why unique name does not work
Also, the idea that in the future it could be replaced via incremental counter does not work for ID preservation. In the future I would think that this needs to be revisited to ensure ID preservation